src/BlazorUI/Bit.BlazorUI/Components/Navs/Pivot/BitPivot.scss

```
@@ -196,19 +226,73 @@
+    }
+}
+
+// Stretches the items over the whole header, which is what turns a handful of tabs into a segmented
+// control rather than a row of labels huddled at one edge.
+.bit-pvt-fwd {
+    // The overflow More button carries .bit-pvti as well, and it is a fixed affordance rather than
+    // one of the tabs sharing the line, so it keeps its natural width.
+    .bit-pvt-hct > .bit-pvti:not(.bit-pvt-mor) {
+        flex: 1 1 0;
+        min-width: 0;
+        // Sharing the whole line leaves nothing between the items unless the pivot asks for a gap.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)